src/scipp/html/formatting_datagroup_html.py

```diff
@@ -42,13 +42,9 @@ def _format_dictionary_item(name_item: tuple, maxidx: int = 10) -> str:
 
 def _format_multi_dim_data(var: Union[Variable, DataArray, Dataset, np.ndarray]) -> str:
     """Inline preview of single or multi-dimensional data"""
-    if isinstance(var, Variable):
-        if var.ndim != var.values.ndim:
-            return _format_atomic_value(var.values, maxidx=None)
-        elif var.ndim == 0:
-            return _format_atomic_value(var.value, maxidx=30)
-
-    if isinstance(var, Dataset):
+    if isinstance(var, (Variable, DataArray)):
+        return inline_variable_repr(var).replace("<div>", '').replace("</div>", '')
```

Comment:
What if there is a string value that contains this? Can you extract the part of `inline_variable_repr` before adding the HTML divs to a new function?

Reply:
That is a very good point... I will just slice the string instead of using `replace`

Reply:
I ended up updating the `formatting_html` to wrap the `inline_variable_repr` with `<div>` tag outside of the method itself.